include/drivers/clock_control/stm32_clock_control.h

```diff
@@ -19,10 +19,10 @@ enum {
 	STM32_CLOCK_BUS_AHB1,
 	STM32_CLOCK_BUS_AHB2,
 	STM32_CLOCK_BUS_APB1,
+	STM32_CLOCK_BUS_APB2,
 #ifdef CONFIG_SOC_SERIES_STM32L4X
 	STM32_CLOCK_BUS_APB1_2,
 #endif
-	STM32_CLOCK_BUS_APB2,
 };
 
```

Comment:
I'm able to #include <cloks.h>
I think it'd be better to #include <dts/arm/st/clocks.h>, but I can't get it work
Any idea how I could do that?